src/Form/CarFormType.php

```diff
@@ -31,13 +31,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('registrationDate', DateType::class, [
                 'widget' => 'single_text', 'required' => true
             ])
-            ->add('user', EntityType::class, [
-                'class' => User::class,
-                'choice_label' => 'email', 
-                'label' => 'Select User',
-                'placeholder' => 'Choose a user', 
-                'required' => true, 
-            ])
+            // ->add('user', EntityType::class, [
```

Comment:
why this is commented?

Reply:
Because you said that there is no need to set user, because every person who is logged in can create,update or delete car but car is his, and there is no need to change the owner